docs/_integration-with-existing-apps-ios.md

```diff
@@ -0,0 +1,596 @@
+    window?.rootViewController = ViewController()
+    window?.makeKeyAndVisible()
+    return true
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Update the `AppDelegate` class
+
+First, we need to extends the `AppDelegate` to inherit from one of the classes provided by React Native: `RCTAppDelegate`.
+
+<Tabs groupId="ios-language" queryString defaultValue={constants.defaultAppleLanguage} values={constants.appleLanguages}>
+<TabItem value="objc">
+
+To achieve this, we have to modify the `AppDelegate.h` file and the `AppDelegate.m` files:
+
+1. Open the `AppDelegate.h` files and modify it as it follows (See the official template's [AppDelegate.h](https://github.com/react-native-community/template/blob/0.76-stable/template/ios/HelloWorld/AppDelegate.h) as reference):
+
+```diff title="AppDelegate.h changes"
+#import <UIKit/UIKit.h>
++#import <React-RCTAppDelegate/RCTAppDelegate.h>
+
+-@interface AppDelegate : UIResponder <UIApplicationDelegate>
++@interface AppDelegate : RCTAppDelegate
+
+
+@end
+```
+
+2. Open the `AppDelegate.mm` file and modify it as it follows (See the official template's [AppDelegate.mm](https://github.com/react-native-community/template/blob/0.76-stable/template/ios/HelloWorld/AppDelegate.mm) as reference
+
+```diff title="AppDelegate.mm"
+#import "AppDelegate.h"
+#import "ViewController.h"
++#import <React/RCTBundleURLProvider.h>
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate {
+  UIWindow *window;
+}
+
+ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
++ self.automaticallyLoadReactNativeWindow = NO;
++ return [super application:application didFinishLaunchingWithOptions:launchOptions];
+   window = [UIWindow new];
+   window.rootViewController = [ViewController new];
+   [window makeKeyAndVisible];
+   return YES;
+
+ }
+
++- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
++{
++  return [self bundleURL];
++}
+
++- (NSURL *)bundleURL
++{
++#if DEBUG
++  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
++#else
++  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
++#endif
++}
+ @end
+```
+
+Let's have a look at the code above:
+
+1. We are inherithing from the `RCTAppDelegate` and we are calling the `applicatinon:didFinishLaunchingWithOptions` of the `RCTAppDelegate`. This delegates all the React Native initialization processes to the base class.
+2. We are customizing the RCTAppDelegate by setting the `automaticallyLoadReactNativeWindow` to `NO`. This step instruct React Native that the app is handling the `UIWindow` and React Native should not worry about that.
+3. The methods `sourceURLForBridge:` and `bundleURL` are used by the App to tell to React Native wher it can find the JS bundle that needs to be rendered. The `sourceURLForBridge:` is from the Old Architecture and you can see that it is deferring the decision to the `bundleURL` method, required by the New Architecture.
```

Comment:
```suggestion
3. The methods `sourceURLForBridge:` and `bundleURL` are used by the App to tell React Native where it can find the JS bundle that needs to be rendered. The `sourceURLForBridge:` is from the Old Architecture and you can see that it is deferring the decision to the `bundleURL` method, required by the New Architecture.
```